portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +35,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     
-  ArrayList<String> comments = new ArrayList<String>();
+  //ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Remember, you can just delete this instead of commenting it out.

(I know old habits die hard.)